iot_verifier/src/loader.rs

```diff
@@ -372,9 +376,7 @@ impl Loader {
                                         report_type: ReportType::Witness,
                                         status: LoraStatus::Ready,
                                     };
-                                    metrics::increment_counter!(
-                                        "oracles_iot_verifier_loader_witness"
-                                    );
+                                    metrics.increment_witnesses();
```

Comment:
Witnesses are being incremented here but not being decremented afaics

Reply:
That is actually to increment the witness counter, so we never need to decrement.  maybe a better function name will eliminate the confusion?

Reply:
Ah that's fine then. Reading on my phone so missed that